tardis/transport/montecarlo/base.py

```diff
@@ -213,6 +213,11 @@ def run(
         # Condition for Checking if RPacket Tracking is enabled
         if self.montecarlo_configuration.ENABLE_RPACKET_TRACKING:
             transport_state.rpacket_tracker = rpacket_trackers
+        else:
+            self.transport_state.rpacket_last_interaction_trackers = (
+                rpacket_last_interaction_trackers
+            )
```

Comment:
@wkerzendorf, should I add a flag for the last interaction tracking class in this PR as suggested and remove that in the upcoming PRs where the last interaction class is actually being used in the simulation?

Reply:
@AlexHls, I have added a flag.

Reply:
If packets are not being tracked properly - at least the last interaction should be tracked - so that is the minimum that should happen. I think for now @Sumit112192 has done a good job with the else statement.